failed to run custom build command for `wabt-sys v0.7.2`

when run `cargo build --release`, error occur:
```
error: failed to run custom build command for `wabt-sys v0.7.2`

Caused by:
  process didn't exit successfully: `/home/wjy/桌面/substrate-2.0.0-rc6/target/release/build/wabt-sys-289be05fb54f66e4/build-script-build` (exit code: 101)
--- stdout
cargo:rerun-if-env-changed=WABT_CXXSTDLIB
cargo:rerun-if-env-changed=CXXSTDLIB
running: "cmake" "/home/wjy/.cargo/registry/src/crates.rustcc.com-a21e0f92747beca3/wabt-sys-0.7.2/wabt" "-DBUILD_TESTS=OFF" "-DBUILD_TOOLS=OFF" "-DCMAKE_INSTALL_PREFIX=/home/wjy/\xe6\xa1\x8c\xe9\x9d\xa2/substrate-2.0.0-rc6/target/release/build/wabt-sys-2248b7ea9d81edf8/out" "-DCMAKE_C_FLAGS= -ffunction-sections -fdata-sections -fPIC -m64" "-DCMAKE_C_COMPILER=/usr/bin/cc" "-DCMAKE_CXX_FLAGS= -ffunction-sections -fdata-sections -fPIC -m64" "-DCMAKE_CXX_COMPILER=/usr/bin/c++" "-DCMAKE_ASM_FLAGS= -ffunction-sections -fdata-sections -fPIC -m64" "-DCMAKE_ASM_COMPILER=/usr/bin/cc" "-DCMAKE_BUILD_TYPE=Release"
-- Configuring done
-- Generating done
-- Build files have been written to: /home/wjy/桌面/substrate-2.0.0-rc6/target/release/build/wabt-sys-2248b7ea9d81edf8/out/build
running: "cmake" "--build" "." "--config" "Release" "--"
[  1%] Building CXX object CMakeFiles/wabt.dir/src/option-parser.cc.o
CMakeFiles/wabt.dir/build.make:686: recipe for target 'CMakeFiles/wabt.dir/src/option-parser.cc.o' failed
CMakeFiles/Makefile2:136: recipe for target 'CMakeFiles/wabt.dir/all' failed
Makefile:83: recipe for target 'all' failed

--- stderr
/home/wjy/.cargo/registry/src/crates.rustcc.com-a21e0f92747beca3/wabt-sys-0.7.2/wabt/src/option-parser.cc: In lambda function:
/home/wjy/.cargo/registry/src/crates.rustcc.com-a21e0f92747beca3/wabt-sys-0.7.2/wabt/src/option-parser.cc:60:20: error: ‘CMAKE_PROJECT_VERSION’ was not declared in this scope
     printf("%s\n", CMAKE_PROJECT_VERSION);
                    ^~~~~~~~~~~~~~~~~~~~~
make[2]: *** [CMakeFiles/wabt.dir/src/option-parser.cc.o] Error 1
make[1]: *** [CMakeFiles/wabt.dir/all] Error 2
make: *** [all] Error 2
thread 'main' panicked at '
command did not execute successfully, got: exit code: 2

build script failed, must exit now', /home/wjy/.cargo/registry/src/crates.rustcc.com-a21e0f92747beca3/cmake-0.1.44/src/lib.rs:885:5
note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace

warning: build failed, waiting for other jobs to finish...
^C  Building [====================================>                  ] 664/970: rustls, bindgen, js-sys 
```

--- a/bin/node/executor/Cargo.toml
+++ b/bin/node/executor/Cargo.toml
@@ -41,7 +41,7 @@ sp-application-crypto = { version = "2.0.0-rc6", path = "../../../primitives/app
 sp-runtime = { version = "2.0.0-rc6", path = "../../../primitives/runtime" }
 sp-externalities = { version = "0.8.0-rc6", path = "../../../primitives/externalities" }
 substrate-test-client = { version = "2.0.0-rc6", path = "../../../test-utils/client" }
-wabt = "0.9.2"
+wabt = "0.9.1"
 
 [features]
 wasmtime = [

--- a/bin/node/testing/Cargo.toml
+++ b/bin/node/testing/Cargo.toml
@@ -39,7 +39,7 @@ substrate-test-client = { version = "2.0.0-rc6", path = "../../../test-utils/cli
 pallet-timestamp = { version = "2.0.0-rc6", path = "../../../frame/timestamp" }
 pallet-transaction-payment = { version = "2.0.0-rc6", path = "../../../frame/transaction-payment" }
 pallet-treasury = { version = "2.0.0-rc6", path = "../../../frame/treasury" }
-wabt = "0.9.2"
+wabt = "0.9.1"
 sp-api = { version = "2.0.0-rc6", path = "../../../primitives/api" }
 sp-finality-tracker = { version = "2.0.0-rc6", default-features = false, path = "../../../primitives/finality-tracker" }
 sp-timestamp = { version = "2.0.0-rc6", default-features = false, path = "../../../primitives/timestamp" }

--- a/client/executor/Cargo.toml
+++ b/client/executor/Cargo.toml
@@ -37,7 +37,7 @@ libsecp256k1 = "0.3.4"
 
 [dev-dependencies]
 assert_matches = "1.3.0"
-wabt = "0.9.2"
+wabt = "0.9.1"
 hex-literal = "0.3.1"
 sc-runtime-test = { version = "2.0.0-rc6", path = "runtime-test" }
 substrate-test-runtime = { version = "2.0.0-rc6", path = "../../test-utils/runtime" }

--- a/primitives/sandbox/Cargo.toml
+++ b/primitives/sandbox/Cargo.toml
@@ -20,7 +20,7 @@ sp-wasm-interface = { version = "2.0.0-rc6", default-features = false, path = ".
 codec = { package = "parity-scale-codec", version = "1.3.1", default-features = false }
 
 [dev-dependencies]
-wabt = "0.9.2"
+wabt = "0.9.1"
 assert_matches = "1.3.0"
 
 [features]

